packages/wallet/backend/src/rapyd/controller.ts

```diff
@@ -81,6 +86,21 @@ export class RapydController implements IRapydController {
       req.session.user.needsWallet = false
       await req.session.save()
 
+      const defaultAccount =
+        await this.deps.accountService.createDefaultAccount(id)
+      if (defaultAccount) {
+        const typedArray = new Uint32Array(1)
```

Comment:
I think create default payment pointer also can be a separate method in payment pointer service

Reply:
I don't think it requires a different method in the payment pointer service.